rekcurd/core/rekcurd_dashboard_servicer.py

```diff
@@ -198,19 +198,10 @@ def EvaluationResult(self,
         data_path = request.data_path
         result_path = request.result_path
         local_data_path = self.rekcurd_pack.app.data_server.get_evaluation_data_path(data_path)
-        local_result_summary_path = self.rekcurd_pack.app.data_server.get_eval_result_summary(result_path)
         local_result_detail_path = self.rekcurd_pack.app.data_server.get_eval_result_detail(result_path)
 
-        with open(local_result_summary_path, 'rb') as f:
-            result = pickle.load(f)
-        label_ios = [self.get_io_by_type(l) for l in result.label]
-        metrics = rekcurd_pb2.EvaluationMetrics(num=result.num,
-                                                accuracy=result.accuracy,
-                                                precision=result.precision,
-                                                recall=result.recall,
-                                                fvalue=result.fvalue,
-                                                option=result.option,
-                                                label=label_ios)
+        # TODO: deprecated. remove metrics from response in the next gRPC spec
+        metrics = rekcurd_pb2.EvaluationMetrics()
```

Comment:
How do you get `EvaluationMetrics` at the beginning?
To save the evaluation result on `dashboard`, we need to get scores from `Rekcurd`.

Reply:
at the beginning, [EvaluateModel method](https://github.com/rekcurd/rekcurd-python/pull/46/files#diff-30a140cb6fa645838e4756f0ddf39de9R165) get & returns EvaluationMetrics to dashboard, and dashboard save the metrics.

This part is just to restore the result of EvaluateModel method.

Reply:
Do you mean we don't need to pack `EvaluationMetrics` to `EvaluationResultResponse`?
I see.